apps/server/src/challenges/dto/challenges.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export type visibilityOptions = 'private' | 'public';
+
+export class CreateChallengeDto {
```

Comment:
This should be in `create-challenges.dto.ts` file.

Reply:
Done